README.md

```diff
@@ -6,66 +6,62 @@
 pymbar
 ======
 
-Python implementation of the multistate Bennett acceptance ratio (MBAR) method for estimating expectations and free energy differences.  See our [Docs](http://pymbar.readthedocs.org/en/latest/).
+Python implementation of the [multistate Bennett acceptance ratio (MBAR)](http://www.alchemistry.org/wiki/Multistate_Bennett_Acceptance_Ratio) method for estimating expectations and free energy differences from equilibrium samples from multiple probability densities.
+See our [docs](http://pymbar.readthedocs.org/en/latest/).
 
 
 Installation
 ------------
 
-The easiest way to install `pymbar` is via [conda](http://conda.pydata.org), a binary package installer that comes with the [Anaconda Scientific Python distribution](https://store.continuum.io/cshop/anaconda/):
-```tcsh
-conda install -c https://conda.binstar.org/omnia pymbar
+The easiest way to install the `pymbar` release is via [conda](http://conda.pydata.org):
+```bash
+conda install -c omnia pymbar
+```
+You can also install `pymbar` from the [Python package index](https://pypi.python.org/pypi/pymbar) using `pip`:
+```bash
+pip install pymbar
+```
+The development version can be installed directly from github via `pip`:
+```bash
+pip install git+https://github.com/choderalab/pymbar.git
 ```
-
-Besides conda, you can also install `pymbar` using `pip` (`pip install pymbar`) or directly from the source directory (`python setup.py install`).
-
 
 Usage
 -----
 
-Basic usage involves importing pymbar, loading data, and constructing an MBAR object from the reduced potential of simulation or experimental data:
+Basic usage involves importing `pymbar` and constructing an `MBAR` object from the [reduced potential](http://www.alchemistry.org/wiki/Multistate_Bennett_Acceptance_Ratio#Reduced_potential) of simulation or experimental data.
 
+Suppose we sample a 1D harmonic oscillator from a few thermodynamic states:
 ```python
->>> from pymbar import MBAR, testsystems
->>> x_k, u_kn, N_k, s_n = testsystems.HarmonicOscillatorsTestCase().sample()
->>> mbar = MBAR(u_kn, N_k)
+>>> from pymbar import testsystems
+>>> [x_n, u_kn, N_k, s_n] = testsystems.HarmonicOscillatorsTestCase().sample()
 ```
+We have the `nsamples` sampled oscillator positions `x_n` (with samples from all states concatenated), [reduced potentials](http://www.alchemistry.org/wiki/Multistate_Bennett_Acceptance_Ratio#Reduced_potential) in the `(nstates,nsamples)` matrix `u_kn`, number of samples per state in the `nsamples` array `N_k`, and indices `s_n` denoting which thermodynamic state each sample was drawn from.
 
-Next, we extract the dimensionless free energy differences and uncertainties:
-
+To analyze this data, we first initialize the `MBAR` object:
+```python
+>>> mbar = MBAR(u_kn, N_k)
+```
+Estimating dimensionless free energy differences between the sampled thermodynamic states and their associated uncertainties (standard errors) simply requires a call to `getFreeEnergyDifferences()`:
```

Comment:
I would add something like "where the free energy differences are relative to the first state" to explicitly state that index 0 is the ref state.